### ISSUE TITLE
add canonical url paths to list details modals/pages

### ISSUE THREAD

**Issue by collinbarrett** [collinbarrett/FilterLists#263]
Unique canonical URL paths should lead to each list details modal and/or page. i.e., `https://filterlists.com/1hosts` should lead to the opened details modal or a separate page with the details of that list. This would improve accessibility, make links sharable, and improve SEO.

related to #250

**Comment by collinbarrett** [comment]
> Hypothetical scenario: Let's say I receive an issue report for my list(s), by an enthusiast who desperately wants me to add filters for cookie notifications, which is a topic that my lists only partially aim to deal with.
> 
> As it stands now, I'd have directed him/her to check out the _I Don't Care About Cookies_ list. But to pre-emptively help prevent further complaints from him/her that the list doesn't work or something like that, then it could've been convenient if I could've linked to all cookie lists at the same time, i.e. all of FilterLists' cookie-tagged lists.
> 
> So I propose to make it possible to add extra values at the end of the site URL, e.g. `https://filterlists.com/?tag=cookies` or `https://filterlists.com/?lang=pl`.

via #406

**Comment by collinbarrett** [comment]
support links:
https://spectrum.chat/thread/ea9b94dc-6291-4a61-99f7-69af4094e90c
https://spectrum.chat/thread/35ca250f-5ea7-48ce-933b-dd58241678e0

**Comment by DandelionSprout** [comment]
Now you can support this Wish from https://beerpay.io/collinbarrett/FilterLists/wishes/5bcc5f715c3b5105c06531ed

🍻 Cheers!

**Comment by collinbarrett** [comment]
This is now partially supported (for lists). I hope to add for taxonomies soon.

**Comment by hawkeye116477** [comment]
Similar feature I made for https://majkiit.github.io/polish-ads-filter/en/ - chosen categories, languages, adblocker, mirror and search keyword are saved on URL parameters :smile: .
